checks/encoding.py

```diff
@@ -115,5 +119,18 @@ def checkEncoding(lines):
             severity = LEVEL_WARNING
         else:
             severity = LEVEL_INFO
-        return [severity, "{}% Encoder Overload".format(val),
-                """Encoder overload may be related to your CPU or GPU being overloaded, depending on the encoder in question. If you are using a software encoder (x264) please see the <a href="https://obsproject.com/wiki/General-Performance-and-Encoding-Issues">CPU Overload Guide</a>. If you are using a hardware encoder (AMF, QSV/Quicksync, NVENC) please see the <a href="https://obsproject.com/wiki/GPU-overload-issues">GPU Overload Guide</a>."""]
+
+        if ((len(hasx264) + len(hasAMD) + len(hasQSV) + hasNVENC) > 1):
+            return [severity, "{}% Encoder Overload".format(val),
+                    """Encoder overload may be related to your CPU or GPU being overloaded, depending on the encoder in question. If you are using a software encoder (x264) please see the <a href="https://obsproject.com/wiki/General-Performance-and-Encoding-Issues">CPU Overload Guide</a>. If you are using a hardware encoder (AMF, QSV/Quicksync, NVENC) please see the <a href="https://obsproject.com/wiki/GPU-overload-issues">GPU Overload Guide</a>."""]
```

Comment:
Would it be worth turning this string into a variable above so that it's clearer that the same text is displayed in both cases?